components/organisation-unit-tree/types/index.d.ts

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react'
+
+export type OrganisationUnitTreeRoots = string | string[]
+
+interface OrganisationUnitNode {
+    id: string
+    displayName: string
+    children?: number
+    path: string
+}
+
+interface OrganisationUnitEventPayload extends OrganisationUnitNode {
+    checked: boolean
+    selected: string[]
+}
+
+export interface NodeLabelProps {
+    hasChildren: boolean
+    hasSelectedDescendants: boolean
+    loading: boolean
+    error: boolean
+    selected: string[]
+    open: boolean
+    path: string[]
+    singleSelection: boolean
+    node: OrganisationUnitNode
+    label: string
+    checked: boolean
+    highlighted: boolean
+}
+
+export interface OrganisationUnitTreeProps {
```

Comment:
Worth a look: Quite complex and a lot of manual work to resolve what types all callbacks are. 
@Mohammer5